examples/door-lock-app/door-lock-common/door-lock-app.matter

```diff
@@ -1531,8 +1772,10 @@ endpoint 0 {
 }
 
 endpoint 1 {
+  server  cluster ColorControl;
```

Comment:
Color control cluster here: this claims endpoint 1 has color control capability. This sounds as unintended.

Reply:
I removed this